src/System.Threading.Tasks/tests/CESchedulerPairTests.cs

```diff
@@ -138,10 +138,10 @@ public static void TestCreationOptions(String ctorType)
                 //In the current design, when there are no more tasks to execute, the Task used by concurrentexclusive scheduler dies
                 //by sleeping we simulate some non trival work that takes time and causes the concurrentexclusive scheduler Task 
                 //to stay around for addition work.
-                taskList.Add(readers.StartNew(() => { new ManualResetEvent(false).WaitOne(10); }));
+                taskList.Add(readers.StartNew(() => { var sw = new SpinWait(); while (!sw.NextSpinWillYield) sw.SpinOnce() ; }));
```

Comment:
It's equivalent to a number of a calls to Thread.SpinWait, which isn't exposed.
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Threading/SpinWait.cs#L167